mesonbuild/backend/ninjabackend.py

```diff
@@ -93,7 +93,7 @@ def gcc_rsp_quote(s):
 # variables (or variables we use them in) is interpreted directly by ninja
 # (e.g. the value of the depfile variable is a pathname that ninja will read
 # from, etc.), so it must not be shell quoted.
-raw_names = {'DEPFILE_UNQUOTED', 'DESC', 'pool', 'description'}
+raw_names = {'DEPFILE_UNQUOTED', 'DESC', 'pool', 'description', 'targetdep'}
```

Comment:
The commentary doesn't explain why this variable fits into the category outlined in the comment above.  Does it? Perhaps the reasoning in the comment is incorrect? Or this change just makes things work? :)

Reply:
You removed it by accident when you made raw_names a global, I think.
I just put it back because otherwise the rust tests all fail on rust's 'ninja -t clean' rules.
It was added by 49e9e017.

Reply:
I disappeared the bug, and added a line in the patch commentary about what breaks if you remove that raw name.